src/components/templates/PartyMap/components/ZoomControls/ZoomControls.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from "react";
+
+interface ZoomControlsProps {
+  disableZoomIn: boolean;
+  disableZoomOut: boolean;
+  onZoomIn: () => void;
+  onZoomOut: () => void;
+}
+
+export const ZoomControls: FC<ZoomControlsProps> = ({
+  disableZoomIn,
+  disableZoomOut,
+  onZoomIn,
+  onZoomOut,
```

Comment:
`disableZoomIn`/`disableZoomOut` could also potentially be wrapped into `onZoomIn`/`onZoomOut` functions so that they are a noop if called (in addition to setting the `disabled` class as you are here)